ledger/ledgercore/totals_test.go

```diff
@@ -206,7 +206,7 @@ var uniqueAccountTotals = []AccountTotals{
 func TestAccountTotalsMarshalMsgUnique(t *testing.T) {
 	partitiontest.PartitionTest(t)
 
-	uniqueAt := make(map[crypto.Digest]bool, 0)
+	uniqueAt := make(map[crypto.Digest]bool)
```

Comment:
If you're already chamgimg this is, why not to size the map to len(uniqueAccountTotals) ?

Reply:
done